<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>repeaterSetup</class>
 <widget class="QMainWindow" name="repeaterSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Repeater Setup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="rptGrpbox">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Repeater Duplex</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="rptSimplexBtn">
           <property name="text">
            <string>Simplex</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptDuplexBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rptDupPlusBtn">
           <property name="text">
            <string>Dup+</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptDuplexBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rptDupMinusBtn">
           <property name="text">
            <string>Dup-</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptDuplexBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rptAutoBtn">
           <property name="text">
            <string>Auto</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptDuplexBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="rptrOffsetSetBtn">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the repeater offset for radios using Repeater modes. Only available on radios that have repeater modes. Radios using Split should instead use the provided Split Mode section with a custom Offset. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Set Offset (MHz):</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="rptrOffsetEdit">
           <property name="placeholderText">
            <string>Rpt Offset (MHz)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="splitModeGrpbox">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Split Mode</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="splitEnableChk">
             <property name="toolTip">
              <string>Turn on Split</string>
             </property>
             <property name="text">
              <string>Split On</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">rptDuplexBtns</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="splitOffBtn">
             <property name="toolTip">
              <string>Turn off Split</string>
             </property>
             <property name="text">
              <string>Split Off</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">rptDuplexBtns</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="quickSplitChk">
             <property name="text">
              <string>QuickSplit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QCheckBox" name="setSplitRptrToneChk">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click here to automatically set the sub VFO (transmit VFO) tone. Only available on some radios. Other radios may take care of this for you.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Set Rpt Tone</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoTrackLiveBtn">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click here to continually set the transmit VFO to match the receive VFO with the offset accounted for.  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>AutoTrack</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Offset (KHz):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="splitOffsetEdit">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Enter the desired split offset in KHz.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="splitPlusButton">
             <property name="maximumSize">
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the transmit frequency to the receive frequency PLUS the offset. Sets the radio sub VFO and also populates the wfview text box (as a convenience). &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Split+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="splitMinusBtn">
             <property name="maximumSize">
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the transmit frequency to the receive frequency PLUS the offset. Sets the radio sub VFO and also populates the wfview text box (as a convenience). &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Split-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Tx Freq (MHz):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="splitTransmitFreqEdit">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="splitTxFreqSetBtn">
             <property name="maximumSize">
              <size>
               <width>116</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the transmit frequency manually. Not needed if the Split+ or Split- button was used. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>VFO</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QPushButton" name="swapABBtn">
           <property name="toolTip">
            <string>Swap VFO A with VFO B. Some radios do not support this.</string>
           </property>
           <property name="text">
            <string>Swap AB</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="selSubBtn">
           <property name="toolTip">
            <string>Select the Sub VFO</string>
           </property>
           <property name="text">
            <string>Sel Sub</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="selMainBtn">
           <property name="toolTip">
            <string>Select the Main VFO</string>
           </property>
           <property name="text">
            <string>Sel Main</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="selBBtn">
           <property name="toolTip">
            <string>Select VFO B</string>
           </property>
           <property name="text">
            <string>Sel B</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="selABtn">
           <property name="toolTip">
            <string>Select VFO A</string>
           </property>
           <property name="text">
            <string>Sel A</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="aEqBBtn">
           <property name="toolTip">
            <string>Set VFO B to VFO A</string>
           </property>
           <property name="text">
            <string>A=B</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="mEqSBtn">
           <property name="toolTip">
            <string>Set the SUB VFO to match the Main VFO</string>
           </property>
           <property name="text">
            <string>M=&gt;S</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="swapMSBtn">
           <property name="toolTip">
            <string>Swap the Main VFO and Sub VFO</string>
           </property>
           <property name="text">
            <string>Swap MS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Repeater Tone Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="warningFMLabel">
           <property name="text">
            <string>Only available in FM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toneNone">
           <property name="text">
            <string>None</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptToneBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toneTone">
           <property name="text">
            <string>Transmit Tone only</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptToneBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toneTSQL">
           <property name="text">
            <string>Tone Squelch</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptToneBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toneDTCS">
           <property name="text">
            <string>DTCS</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rptToneBtns</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setRptrSubVFOBtn">
           <property name="toolTip">
            <string>Set the Tone Mode for the Sub VFO. Not available on all radios.</string>
           </property>
           <property name="text">
            <string>Set Sub VFO</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Tone Selection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Tone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rptToneCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>TSQL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rptTSQLCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>DTCS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rptDTCSCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="rptDTCSInvertTx">
             <property name="text">
              <string>Invert Tx</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="rptDTCSInvertRx">
             <property name="text">
              <string>Invert Rx</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="setToneSubVFOBtn">
           <property name="toolTip">
            <string>Set the Sub VFO to the selected Tone. Not available on all radios and not available for DTCS.</string>
           </property>
           <property name="text">
            <string>Set Sub VFO</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rptToneBtns"/>
  <buttongroup name="rptDuplexBtns"/>
 </buttongroups>
</ui>
